src/style/sections/exports/exports.less

```diff
@@ -165,6 +165,33 @@
     flex-direction: row;
 }
 
+.layer-exports__header {
+   width: 100%;
+
+   .formline.formline__space-between {
```

Comment:
It feels like (which is a terrible phrase to use) that there is a cleaner way to do this...I'm looking into it